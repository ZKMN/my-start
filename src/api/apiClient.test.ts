import apiClient from "./apiClient";

describe("apiClient", () => {
  it("apiClient", () => {
    expect(apiClient.get("")).toMatchSnapshot();
  });
});
