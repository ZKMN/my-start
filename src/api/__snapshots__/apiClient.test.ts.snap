// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`apiClient apiClient 1`] = `Promise {}`;
